src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
Given our current uses of simulate, it's true that creating instantiation would not have substantial performance impact. But I still don't see the value in creating an object with no useful properties. How is `jQuery.Event{ type: undefined, timeStamp: <Date>, [jQuery.expando]: true }` better than `{}`, and how is `{}` better than `undefined`?